samza-core/src/main/scala/org/apache/samza/coordinator/JobModelManager.scala

```diff
@@ -156,7 +156,7 @@ object JobModelManager extends Logging {
     jobModelRef.set(jobModel)
 
     val server = new HttpServer
-    server.addServlet("/*", new JobServlet(jobModelRef))
+    server.addServlet("/", new JobServlet(jobModelRef))
```

Comment:
the `*` matches all paths from the root to the servlet. since we have a new endpoint this needs to be removed.